.gitignore

```diff
@@ -2,8 +2,9 @@ __pycache__
 *.egg-info
 build
 .DS_STORE
-comfyui*
```

Comment:
@ryanontheinside was there a reason to use the wildcard or can we simply ignore the ComfyUI folder?